queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -126,13 +131,13 @@ def process(business: Business, filing: Dict, filing_rec: Filing):
     if parties := incorp_filing.get('parties'):
         update_parties(business, parties)
 
-    if share_structure := incorp_filing['shareStructure']:
+    if share_structure := incorp_filing.get('shareStructure'):
         shares.update_share_structure(business, share_structure)
 
     if name_translations := incorp_filing.get('nameTranslations'):
         aliases.update_aliases(business, name_translations)
 
-    if not is_correction:
+    if not is_correction and not filing_rec.colin_event_ids:
         # Update the filing json with identifier and founding date.
         ia_json = copy.deepcopy(filing_rec.filing_json)
         ia_json['filing']['business']['identifier'] = business.identifier
```

Comment:
just curious if we need to copy anything else from colin (like founding date), or is this done elsewhere?

Reply:
yes, that will be the filings effective date, for setting the founding date of the existing corp

Reply:
gotcha